test/deploy/k8s/cortx-storagenode-statefulset.yml

```diff
@@ -0,0 +1,94 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: storagenode
+  labels:
+    app: storagenode
+spec:
+  serviceName: "storageservice"
+  replicas: 3
+  selector:
+    matchLabels:
+      app: storagenode
+  template:
+    metadata:
+      labels:
+        app: storagenode
+    spec:
+      volumes:
+      - name: cortx-config
+        persistentVolumeClaim:
+          claimName: cortx-config-pvc
+      - name: solution-config
+        configMap:
+          name: solution-config
+      - name: cortx-secret
+        secret:
+          secretName: cortx-secret
+      - name: node-info
+        downwardAPI:
+          items:
+          - path: "name"
+            fieldRef:
+              fieldPath: metadata.name
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - topologyKey: kubernetes.io/hostname
+            labelSelector:
+              matchLabels:
+                app: storagenode
+      setHostnameAsFQDN: true
```

Comment:
`setHostnameAsFQDN: true` will not work with our standard kubectl version v1.19
Lets set hostname in container directly by copying value from `hostname --fqdn`